src/track/cueinfo.h

```diff
@@ -19,6 +19,10 @@ enum class CueType {
     Outro = 7,
     N60dBSound = 8, // range that covers beginning and end of audible
                     // sound; not shown to user
+    FadeIn = 9,     // First time sound reaches a certain level
+                    // (cf analyzersilence.cpp);  not shown to user
+    FadeOut = 10    // Last time sound reaches a certain level
+                    // (cf analyzersilence.cpp);  not shown to user
```

Comment:
>  A perfect solution would be to use a certain Loudness level after applying replay gain. Did you consider to implement that?

I did, but I was struggling to see how the track could have silence markers and not be normalised.   I haven't quite understood the order of operation, and how often the analysis section is called.  And importantly if normalisation is before or after silence detection.  It seems to be resolved several times as far as I can tell... (mostly from the spurious results I was getting while a track was being loaded).  All of which says this probably needs to be considered well before implementation.  Right now, I'm not sure I know enough.  But I am happy to take a look.  Should this be a discussion in chat, since this is a separate topic?

> Please add also a comment about the level itself.
> Is it the last time of full volume, or just an alternative silence threshold?

It is the last or first time the sound reaches a certain level.  Since full volume is subjective, I thought this was more precise.  It is like an alternative silence threshold but is different because there are two different threshold values.  

 (I guess this isn't clear - so perhaps I should add something like: So to be clear, this is the last sample in the track where the volume was above the FadeOut level, and the first sample in the track where the sound volume was above the FadeIn level.)

Reply:
```
    FadeIn = 9,     // Range from the start of the track to the first sample in
                    // the track where the sound volume was above the
                    // kFadeInThreshold (cf analyzersilence.cpp);
                    // not shown to user
    FadeOut = 10    // Range from the last sample in the track where
                    // the sound volume was above the kFadeOutThreshold
                    // to the end of the track (cf analyzersilence.cpp);
                    // not shown to user
```